api/Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM amsterdam/python:3.8-buster
 MAINTAINER datapunt@amsterdam.nl
 
 ENV PYTHONUNBUFFERED 1
+ENV AUTHORIZATION_TOKEN dev
```

Comment:
Ik zou dit een ARG maken, dan is hij beschikbaar als ENV tijdens de build, maar is de ENV token als je de container draait niet standaard `dev`